packages/lib/src/spatial-navigation/components/virtualizedList/SpatialNavigationVirtualizedList.test.tsx

```diff
@@ -278,4 +315,111 @@ describe('SpatialNavigationVirtualizedList', () => {
     expectButtonToHaveFocus(component, 'button 10');
     expect(listElement).toHaveStyle({ transform: [{ translateX: -700 }] });
   });
+
+  it('handles correctly different item sizes', async () => {
+    const component = render(
+      <SpatialNavigationRoot>
+        <DefaultFocus>
+          <SpatialNavigationVirtualizedList
+            testID="test-list"
+            renderItem={renderItem}
+            data={dataWithVariableSizes}
+            itemSize={itemSize}
+            numberOfRenderedItems={5}
+            numberOfItemsVisibleOnScreen={3}
+          />
+        </DefaultFocus>
+      </SpatialNavigationRoot>,
+    );
+    act(() => jest.runAllTimers());
+
+    fireLayoutEvent(component, 300, 300);
+
+    const listElement = await component.findByTestId(listTestId);
+    expect(listElement).toHaveStyle({ transform: [{ translateX: 0 }] });
+    expect(listElement).toHaveStyle({ width: 1500 });
+
+    testRemoteControlManager.handleRight();
+    act(() => jest.runAllTimers());
```

Comment:
didn't you take this out?

Reply:
Done, didn't do the entire file strangely